mava/systems/jax/mappo/networks.py

```diff
@@ -49,7 +49,15 @@ def __init__(
         entropy: Optional[EntropyFn] = None,
         sample: Optional[networks_lib.SampleFn] = None,
     ) -> None:
-        """TODO: Add description here."""
+        """Initialises the PPO network Class.
+
+        Args:
+            network: feedforward network representing the agent policy function.
```

Comment:
won't we use the same classes for recurrent PPO? Then this would not have to be feedforward

Reply:
Changed to  `neural network`